modules/ossm-configuring-jaeger.adoc

```diff
@@ -1,178 +1,15 @@
-              memory: "16Gi"
-
-----
-
-.Elasticsearch parameters
-[options="header"]
-[cols="l, a, a, a, a"]
-|===
-|Parameter |Description |Values |Default Value |Examples
-
-|tracing:
-  enabled
-|This parameter enables/disables tracing in {ProductShortName}. Jaeger is installed by default.
-|`true`/`false`
-|`true`
-|
-
-|ingress:
-  enabled
-|This parameter enables/disables ingress for Jaeger.
-|`true`/`false`
-|`true`
-|
-
-|jaeger
-   template
-|This parameter specifies which Jaeger deployment strategy to use.
-|`all-in-one`/`production-elasticsearch`
-|`all-in-one`
-|
-
-|elasticsearch:
-  nodeCount
-|Number of Elasticsearch nodes to create.
-|Integer value.
-|1
-|Proof of concept = 1,
-Minimum deployment =3
-
-|requests:
-  cpu
-|Number of central processing units for requests, based on your environment’s configuration.
-|Specified in cores or millicores (for example, 200m, 0.5, 1).
-|1Gi
-|Proof of concept = 500m,
-Minimum deployment =1
-
-|requests:
-  memory
-|Available memory for requests, based on your environment’s configuration.
-|Specified in bytes (for example, 200Ki, 50Mi, 5Gi).
-|500m
-|Proof of concept = 1Gi,
-Minimum deployment = 16Gi*
-
-|limits:
-  cpu
-|Limit on number of central processing units, based on your environment’s configuration.
-|Specified in cores or millicores (for example, 200m, 0.5, 1).
-|
-|Proof of concept = 500m,
-Minimum deployment =1
-
-|limits:
-  memory
-|Available memory limit based on your environment’s configuration.
-|Specified in bytes (for example, 200Ki, 50Mi, 5Gi).
-|
-|Proof of concept = 1Gi,
-Minimum deployment = 16Gi*
-
-|
-4+|{asterisk} Each Elasticsearch node can operate with a lower memory setting though this is *not* recommended for production deployments. For production use, you should have no less than 16Gi allocated to each pod by default, but preferably allocate as much as you can, up to 64Gi per pod.
-|===
-
-
-.Procedure
-
-. Log in to the {product-title} web console as a user with the `cluster-admin` role.
-
-. Navigate to *Operators* -> *Installed Operators*.
-
-. Click the {ProductName} Operator.
-
-. Click the *Istio Service Mesh Control Plane* tab.
-
-. Click the name of your control plane file, for example, `basic-install`.
-
-. Click the *YAML* tab.
-
-. Edit the Jaeger parameters, replacing the default `all-in-one` template with parameters for the `production-elasticsearch` template, modified for your use case.  Ensure that the indentation is correct.
-
-. Click *Save*.
-
-. Click *Reload*.
-{product-title} redeploys Jaeger and creates the Elasticsearch resources based on the specified parameters.
+* Configure and deploy a Jaeger instance and then specify the `name` of the Jaeger resource as the value for `spec:addons:jaeger:name` in the `ServiceMeshControlPlane` resource.  If a Jaeger resource matching the value of `name` exists, the control plane will use the existing installation.  This approach lets you fully customize your Jaeger configuration.
```

Comment:
I'm confused - the changes in this file relate to the 2.x SMCP?

Reply:
The name field does not exist in v1 smcp resource.

Reply:
Yes, sorry, this is for 2.0.   I couldn't get the build to pass without copying some of the 2.0 files into this PR.